### PR TITLE
Added Support for User Defined Ref Num. Allow XY Graph and Waveform Graph to be read as Array

### PR DESCRIPTION
**Pull request by pkthong**
Using the autolv component, we found that some control types would throw errors.
Specifically : 
- the user defined ref num should be an IOControl
- XY Graph is an Array
- Waveform Graph is an Array
- Otherwise, instantiate the control as a NotImplControl type, to allow the VI to run

### CODE REVIEW

File: autolv/datatypes.py [hunk 1]
```diff
@@ -430,14 +430,23 @@ def __repr__(self):
     def __str__(self):
         return f"{self.items[self.value]}"
 
+class NotImplControl(LV_Control):
+    """Control Not Implemented"""
+    def __init__(self, **kwargs):
```
Comment (l-johnston) [inline autolv/datatypes.py]: No need to define `__init__()` in this case and pylint complains.

File: autolv/datatypes.py [hunk 2]
```diff
@@ -430,14 +430,23 @@ def __repr__(self):
...
+        super().__init__(**kwargs)
+
+    def __repr__(self):
+        return f"Not Implemented"
```
Comment (l-johnston) [inline autolv/datatypes.py]: f-string notation not needed in this case and pylint complains.

File: autolv/datatypes.py [hunk 3]
```diff
@@ -430,14 +430,23 @@ def __repr__(self):
...
+
+    def __str__(self):
+        return f"Not Implemented"
```
Comment (l-johnston) [inline autolv/datatypes.py]: same as above

**Review by l-johnston — CHANGES_REQUESTED** [review]
Thanks for the contribution.

bump version from 0.2.9 to 0.3.0 in `setup.cfg` and `__init__.py`

File: autolv/__init__.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
 """
 from autolv.interface import App, FPState
 
-__version__ = "0.2.9"
```
Comment (pkthong) [inline autolv/__init__.py:25]: Now 0.3.0

File: setup.cfg [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = autolv
-version = 0.2.9
```
Comment (pkthong) [inline setup.cfg:3]: Now 0.3.0

File: autolv/datatypes.py [hunk 6]
```diff
@@ -466,8 +464,9 @@ def __str__(self):
...
     control_type = attrs.pop("type")
-    if control_type in LVControl_LU:
-        LV_Control_cls = LVControl_LU[control_type]
-    else:
-        LV_Control_cls = NotImplControl
+    LV_Control_cls = LVControl_LU.get(control_type, NotImplControl)
+    # if control_type in LVControl_LU:
+    #     LV_Control_cls = LVControl_LU[control_type]
+    # else:
+    #     LV_Control_cls = NotImplControl
```
Comment (pkthong) [inline autolv/datatypes.py]: <style> Remove dead code

**Comment by pkthong** [comment]
Ready for sign off

**Review by l-johnston — APPROVED** [review]

**Pull request merged**